buildSrc/src/main/java/com/google/firebase/gradle/plugins/PublishingPlugin.kt

```diff
@@ -147,6 +154,19 @@ abstract class PublishingPlugin : Plugin<Project> {
     }
   }
 
+  protected fun registerLibraryGroupsTaks(
+    project: Project,
+    libraryGroups: Map<String, List<FirebaseLibraryExtension>>
+  ) {
+    project.tasks.register("libraryGroups") {
```

Comment:
Should make this a companion const, similar to the other tasks, so that if another needs to a dependency on it- we're passing out consts versus assuming the name of a task.

```kotlin
const val LIBRARY_GROUPS_TASK = "libraryGroups"
```